.travis.yml

```diff
@@ -2,6 +2,7 @@ language: node_js
 node_js:
   - "8"
 script:
+  - npm run lint
```

Comment:
@acejam I believe you wanted to be included in the PR review for when this line gets added.